tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py

```diff
@@ -218,11 +229,9 @@ def test002_start_stop_vm(self):
         self.wait_for_service_action_status(self.vm1, res[self.vm1]['start'])
 
         self.log(" Check that [VM1] is running.")
-        cloudspaceId = self.get_cloudspace(self.cs1)['id']
-        vm = self.get_vm(cloudspaceId, self.vm1)
-        self.assertEqual(vm["status"], "RUNNING")
+        cloudspaceId = self.get_cloudspace(self.cs1_name)['id']
+        self.assertTrue(self.wait_for_vm_status(cloudspaceId, self.vm1_name))
```

Comment:
where is wait_for_vm_status fun?

Reply:
on G8_testing  framework

Reply:
https://github.com/0-complexity/G8_testing/pull/396/files